python/ray/dashboard/client/src/pages/dashboard/Dashboard.tsx

```diff
@@ -47,26 +55,46 @@ class Dashboard extends React.Component<
     typeof mapDispatchToProps
 > {
   timeoutId = 0;
+  tabs = [
+    { label: "Machine view", component: NodeInfo },
+    { label: "Logical view", component: LogicalView },
+    { label: "Ray config", component: RayConfig },
+    { label: "Memory", component: MemoryInfo },
```

Comment:
This is minor, but I would make the table "Memory View" and put it before the "Ray Config" tab, so that the various resource views stay grouped. If you disagree it's fine, I don't feel strongly.